starter-code/app.js

```diff
@@ -1,14 +1,26 @@
-var express      = require('express');
-var path         = require('path');
-var favicon      = require('serve-favicon');
-var logger       = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser   = require('body-parser');
+/*jshint esversion: 6 */
```

Comment:
instead of adding this to every file you should create a file called `starter-code/.jshintrc` containing

```
{
  "esversion": 6
}
```